src/providers/LedgerWebTransport.js

```diff
@@ -0,0 +1,7 @@
+import WebUSBTransport from '@ledgerhq/hw-transport-webusb'
```

Comment:
What's the purpose of this extra class?

Reply:
It adds extra static member `isWebUSBTransport` to the class, so we can check & run claim/release logic just for the browser (webusb) and not in node (hid).